Makefile

```diff
@@ -6,6 +6,9 @@ PACKAGES_DIR := packages
 PLUGINS_DIR := plugins
 SCRIPTS_DIR := scripts
 AEA_TESTS_DIR := tests
+AEA_CORE_TESTS_DIRS := tests/test_aea tests/test_aea_extra ./tests/test_docs
+EXAMPLES_TESTS_DIRS := tests/test_examples
+PACKAGES_TESTS_DIRS := packages/fetchai/protocols packages/fetchai/connections packages/fetchai/skills ./tests/test_packages ./tests/test_packages_for_aea_tests ./tests/test_aea_core_packages
```

Comment:
Add a separate `DOCS_TESTS_DIR` as well.

Reply:
And a test-docs command below using it.

Reply:
done